libse/Forms/FixCommonErrors/FixContinuationStyle.cs

```diff
@@ -85,6 +105,15 @@ public void Fix(Subtitle subtitle, IFixCallbacks callbacks)
                                     isChecked = false;
                                 }
                             }
+
+                            // ...ignore bold tags for Portuguese
+                            if (callbacks.Language == "pt")
+                            {
+                                if (ContinuationUtilities.IsBold(oldText) || ContinuationUtilities.IsBold(oldText))
```

Comment:
As Codacy asks... is this on purpose?

Reply:
Nope. Strange that ReSharper didn't warn for this...
Fixed now.